### ISSUE TITLE
Request update to pytorch_transformers

### ISSUE THREAD

**Issue by zbloss** [huggingface/transfer-learning-conv-ai#26]
I'm running into multiple errors when trying to use this with torch==1.2.0 and pytorch_transformers.

The most difficult to solve is a RuntimeError which seems to pertain to the difference in implementation of the nn.Embedding layer

`RuntimeError: index out of range: Tried to access index 50257 out of table with 50256 rows. at C:\w\1\s\windows\pytorch\aten\src\TH/generic/THTensorEvenMoreMath.cpp:237`

**Comment by thomwolf** [comment]
Indeed, I won't have time to work on that in the short term.
Do you want to submit a PR to update the repo?

**Comment by kiddyboots216** [comment]
@zbloss can you elaborate what line to add that fix?

**Comment by wise-east** [comment]
@kiddyboots216 

```
    tokenizer = tokenizer_class.from_pretrained(args.model_checkpoint)
    model_class = GPT2DoubleHeadsModel if "gpt2" in args.model_checkpoint else OpenAIGPTDoubleHeadsModel
    model = model_class.from_pretrained(args.model_checkpoint)
    tokenizer.set_special_tokens(SPECIAL_TOKENS)
    model.set_num_special_tokens(len(SPECIAL_TOKENS))
    model.to(args.device)
    optimizer = OpenAIAdam(model.parameters(), lr=args.lr)
```
Has to be changed to: 

```
    tokenizer = tokenizer_class.from_pretrained(args.model_checkpoint)
    model_class = GPT2DoubleHeadsModel if "gpt2" in args.model_checkpoint else OpenAIGPTDoubleHeadsModel
    model = model_class.from_pretrained(args.model_checkpoint)
    num_added_token = tokenizer.add_special_tokens(SPECIAL_TOKENS)
    print("Number of added tokens: ", num_added_token)
    print("Special tokens: ", tokenizer.all_special_tokens)
    model.resize_token_embeddings(len(tokenizer))
    optimizer = AdamW(model.parameters(), lr=args.lr)
```

You have to change some of the imports too, such as OpenAIAdam to AdamW, etc. Check the [migration notes](https://huggingface.co/pytorch-transformers/migration.html)

**Comment by sshleifer** [comment]
Fixed by #27 :)
